### ISSUE TITLE
Accuracy of local models in first federated learning round

### ISSUE THREAD

**Issue by chenxi-z325** [KCL-BMEIS/UltraFlwr#50]
Is the pre-trained weight loaded when the YOLO model is initialized? I found that the accuracy of the local model in the first few epochs of the first federated learning round is very high, which is not achieved in the centralized YOLO training

**Comment by aymuos15** [comment]
Yes, if you run it off-the-shelf, **in the the federated setting** we have opted to start training with pretrained weights as per Ultralytics.

Are you performing your central training with our code? If so, **in the centralised setting** we have set it to not use any pretrained weights.  Maybe we should change that because, in our experiments in the paper, the centralised scores are based on pretrained models. 

If you want the same pretrained weights as Ultralytics please replace:

```python
model = YOLO()
```

in `https://github.com/KCL-BMEIS/UltraFlwr/blob/main/scripts/central_train_and_test/local_train_and_test.py`

to 

```python
model = YOLO("yolo11n.pt")
```

**Comment by chenxi-z325** [comment]
Thanks for your reply.My centralized training was performed using the source code of the Ultralytics library, and no pre-trained weights were loaded. I think if we want to compare federated training and centralized training, it is reasonable not to load pre-trained weights.

**Comment by aymuos15** [comment]
I think it's still fair if both are preloaded with the exact same weights.

Is there something specific I am missing as to why the above would be false?

**Comment by chenxi-z325** [comment]
Yes, it is indeed fair to say that both have pre-trained weights as you said.

**Comment by aymuos15** [comment]
Great!

Will keep this open till the other issue is solved. Thank you very much for the insights.
